submodules/misc/misc.js

```diff
@@ -1619,6 +1801,75 @@ define(function(require) {
 					callback && callback(data.data);
 				}
 			});
+		},
+
+		miscSchemasList: function(callback) {
+			var self = this;
+
+			self.callApi({
+				resource: 'schemas.list',
+				success: function(data, status) {
+					callback && callback(data.data);
+				}
+			});
+		},
+
+		miscGetSchema: function(args) {
+			var self = this,
+				data = args.data;
+
+			self.callApi({
+				resource: 'schemas.get',
+				data: {
+					schemaId: data.schemaId
+				},
+				success: function(data, status) {
+					args.hasOwnProperty('success') && args.success(data.data);
+				},
+				error: function(parsedError) {
+					args.hasOwnProperty('error') && args.error(parsedError);
+				}
+			});
+		},
+
+		/*@param reflist: array of the references to other schemas
+		*@param parentSchema: schema with references to other schemas
+		*@param jsoneditor: function to set schema in jsoneditor whith its references
+		fetch all subschemas of partenSchema so they can be set on the jsoneditor*/
```

Comment:
We use JSDoc format for method descriptions, please try to follow that formatting ([doc](https://jsdoc.app/index.html)) for consistency's sake. Most likely, you can add a plugin to your text editor to auto generate the structure for you.